var/spack/repos/builtin/packages/opendatadetector/package.py

```diff
@@ -32,10 +32,10 @@ class Opendatadetector(CMakePackage):
 
     def cmake_args(self):
         args = []
-        # C++ Standard
         args.append("-DCMAKE_CXX_STANDARD=%s" % self.spec["root"].variants["cxxstd"].value)
         return args
 
     def setup_run_environment(self, env):
+        env.set("OPENDATADETECTOR", self.prefix.share + "/OpenDataDetector")
```

Comment:
Use join path for windows support.

Reply:
Also, I'd say it might be better to call this `OPENDATADETECTOR_DATA` since this is pointing to the FHS data directory and just using `OPENDATADETECTOR` could be interpreted as the plain prefix.

Reply:
Fixed